tests/cold/conftest.py

```
@@ -0,0 +1,79 @@
+    _source_provider.disconnect()
+
+
+@pytest.fixture(scope="class")
+def ca_crt_source_provider_inventory(
+    ocp_admin_client: "DynamicClient",
+    mtv_namespace: str,
+    ca_crt_source_provider: BaseProvider,
+) -> ForkliftInventory:
+    """Create a ForkliftInventory for the ca.crt provider."""
+    if not ca_crt_source_provider.ocp_resource:
+        raise ValueError("ca_crt_source_provider.ocp_resource is not set")
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)